samples_project/Assets/SampleViewer/Samples/Measure/readme.md

```diff
@@ -2,13 +2,19 @@
 
```

Comment:
For the h1 title, I think we better use `# Measure distance`...

Reply:
good point, Adrien and I talked a few days ago and we're likely add measure area to this sample, so maybe better to keep it as measure?

Reply:
If I have more information about the new sample you and Adrien talked about, maybe we have better names... We can talk about it this week.